bundles/org.openhab.binding.nanoleaf/README.md

```diff
@@ -4,25 +4,25 @@ This binding integrates the [Nanoleaf Light Panels](https://nanoleaf.me/en/consu
 
 ![Image](doc/Nanoleaf.jpg)
 
-It enables you to authenticate, control, and obtain information of a Light Panel's device.
+It enables you to authenticate, control, and obtain information from the device.
 The binding uses the [Nanoleaf OpenAPI](https://forum.nanoleaf.me/docs/openapi), which requires firmware version [1.5.0](https://helpdesk.nanoleaf.me/hc/en-us/articles/214006129-Light-Panels-Firmware-Release-Notes) or higher.
 
 ![Image](doc/LightPanels2_small.jpg) ![Image](doc/the-worm-small.png) ![Image](doc/NanoCanvas_small.jpg)
 
 ## Supported Things
 
-Nanoleaf provides a bunch of devices of which some are connected to Wifi whereas other use the new Thread Technology. This binding only supports devices that are connected through Wifi.
+Nanoleaf provides a range of devices, some connected via WiFi while others use the new Thread technology. This binding only supports devices that are connected through WiFi.
```

Comment:
For Netatmo you changed "Wifi" to "Wi-Fi". Just wanted to mention this inconsistency.